cns/hnsclient/hnsclient_windows.go

```diff
@@ -301,10 +305,10 @@ func createHostNCApipaNetwork(
 		if interfaceExists, _ := networkcontainers.InterfaceExists(hostNCLoopbackAdapterName); !interfaceExists {
 			ipconfig := cns.IPConfiguration{
 				IPSubnet: cns.IPSubnet{
-					IPAddress:    localIPConfiguration.GatewayIPAddress,
+					IPAddress:    hnsLoopbackAdapterIPAddress,
```

Comment:
why changing here? anyway you are updating with `updateGwForLocalIPConfiguration`